[Feature] Pipe functions (+ bug fixes) (#5)
marcusolsson/obsidian-pipe-tricks: README.md

- [[Yours, Mine and Ours (1968 film)|]] -> [[Yours, Mine and Ours (1968 film)|Yours, Mine and Ours]]
- [[Il Buono, il Brutto, il Cattivo|]] -> [[Il Buono, il Brutto, il Cattivo|Il Buono]]
- [[pipe (computing)|]]! -> [[pipe (computing)|pipe]]!

## Pipe functions

> Additionally one can easily transform link text using __pipe functions__.

Reviewer: ```suggestion
You can also transform link text using __pipe functions__.

```

> Your links must still end in `|`, but additional `|` before that transforms your text using built-in pipe functions.

Reviewer: ```suggestion
Your links must still end in `|`, but an additional `|` before that transforms your text using built-in pipe functions.

```
